gunagala/tests/test_psf.py

```diff
@@ -83,6 +80,16 @@ def test_n_pix_pix(psf_pixellated):
     assert psf_pixellated.n_pix / u.pixel == pytest.approx(21.069945447)
 
 
+# @pytest.mark.parametrize("psf, expected_peak", [
```

Comment:
What is all the commented out code?

Reply:
Our goal is to use `@pytest.mark.parametrize` in the test but it does not work right now because of some problems related to issue #27 . It means those commented lines will be implemented when that problem is fixed. I submitted this PR before the changes increase so much to confuse people and myself.